### PR TITLE
Keyboard movement in workbench sample logs gui

### PR DESCRIPTION
**Pull request by NickDraper**
**Description of work.**
Changed the sample logs view to update on selectionChanged rather than just clicked

**To test:**

1. in workbench load a file with some logs
1. Show Sample Logs
1. Check that you can both click on a log and also move between them with the cursor keys and the plot and stats update.
1. check that multiple selection still works.

refs #28569

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by PhilColebrooke — CHANGES_REQUESTED** [review]
This works as intended 👍 Just some code comments

File: qt/python/mantidqt/widgets/samplelogs/presenter.py [hunk 1]
```diff
@@ -24,8 +24,14 @@ def __init__(self, ws, parent=None, model=None, view=None):
                                                      self.model.getNumExperimentInfo())
         self.setup_table()
 
-    def clicked(self):
-        """When a log is clicked update stats with selected log and replot the
+    def selectionChanged(self, selected, deselected):
```
[the remark this answers is not in the document]
Comment (NickDraper) [reply qt/python/mantidqt/widgets/samplelogs/presenter.py:27]: I just learned that you don't have to match the arguments of a signal and slot in pyQt 😄

File: qt/python/mantidqt/widgets/samplelogs/presenter.py [hunk 2]
```diff
@@ -24,8 +24,14 @@ def __init__(self, ws, parent=None, model=None, view=None):
...
+        """When the selection is changed update stats with selected log and replot the
+                logs
+                """
+        self.update()
+
+    def update(self):
+        """Update stats with selected log and replot the
```
[the remark this answers is not in the document]
Comment (NickDraper) [reply qt/python/mantidqt/widgets/samplelogs/presenter.py:34]: done

**Review by PhilColebrooke — APPROVED** [review]
Looks good to me 😄

**Pull request merged**